listenbrainz/webserver/templates/index/musicbrainz-offline.html

```diff
@@ -0,0 +1,21 @@
+{%- extends 'base.html' -%}
+{%- block title -%}MusicBrainz currently unavailable - ListenBrainz{%- endblock -%}
+{%- block content -%}
+  <h2 class="page-title">MusicBrainz currently unavailable</h2>
+
+  <p>
+     The MusicBrainz database is currently offline for maintenance. As a result,
```

Comment:
I'd consider making this explanation a bit more generic, it might be confusing to people to read something about MusicBrainz while they're on LB. How about something like "ListenBrainz login is currently unavailable due to database maintenance"

Reply:
Fixed.